src/main/webapp/requestResendLinks.jsp

```diff
@@ -11,7 +11,7 @@
   <h1 class="color-orange">
     Request for Resending of Access Links
   </h1>
-  <p id="message">Please enter your email address, an email containing links to all the feedback sessions that you participated in the recent six months will be resent to you.</p>
+  <p id="message">Please enter your email address; an email containing links to all the feedback sessions that you participated over the recent six months will be sent to you.</p>
```

Comment:
Either: ... sessions that you participated **in** over...
Or: ... sessions **in which** you participated over...